src/include/rbd/librbd.h

```diff
@@ -228,6 +229,48 @@ typedef struct {
   rbd_mirror_image_site_status_t *site_statuses;
 } rbd_mirror_image_global_status_t;
 
+typedef enum {
+  RBD_MIRROR_GROUP_DISABLING = 0,
+  RBD_MIRROR_GROUP_ENABLING = 1,
+  RBD_MIRROR_GROUP_ENABLED = 2,
+  RBD_MIRROR_GROUP_DISABLED = 3
+} rbd_mirror_group_state_t;
+
+typedef struct {
+  char *global_id;
+  rbd_mirror_image_mode_t mirror_image_mode;
+  rbd_mirror_group_state_t state;
+  bool primary;
+} rbd_mirror_group_info_t;
+
+typedef enum {
+  MIRROR_GROUP_STATUS_STATE_UNKNOWN         = 0,
+  MIRROR_GROUP_STATUS_STATE_ERROR           = 1,
+  MIRROR_GROUP_STATUS_STATE_STARTING_REPLAY = 2,
+  MIRROR_GROUP_STATUS_STATE_REPLAYING       = 3,
+  MIRROR_GROUP_STATUS_STATE_STOPPING_REPLAY = 4,
+  MIRROR_GROUP_STATUS_STATE_STOPPED         = 5,
+} rbd_mirror_group_status_state_t;
```

Comment:
are we missing the syncing status similar to `MirrorImageStatusStateSyncing` ?  @pkalever